lib/modules/apostrophe-areas/lib/helpers.js

```diff
@@ -188,6 +188,55 @@ module.exports = function(self, options) {
 
     plaintext: function(within, options) {
       return self.plaintext(within, options);
+    },
+    
+    widgetControlGroups: function() {
```

Comment:
Whoops. 

I originally wrote this branch in workflow-accommodations and then cherry picked it back out to master.. i missed this conflict.. I think tooltips should just go into master for now, yeah?

I reverted this file back to its `master` branch state